.gitignore

```diff
@@ -6,6 +6,5 @@ docs/site/
 docs/docs/examples/preview
 .DS_Store
 .env.json
-package-lock.json
 coverage/
-yarn.lock
\ No newline at end of file
+yarn.lock
```

Comment:
why did you choose package-lock.json rather than yarn.lock?

Reply:
The current setup steps use `npm ...` so `package-lock.json` is created by default.

Reply:
i'm concerned about preferring npm on some projects and yarn on others. In my opinion, we should use yarn everywhere. Doesn't need to happen here (although this PR is introducing +10000 new lines due to its inclusion of package-lock.json)

Reply:
I think it's reasonable to standardize on `yarn`. Please keep in mind this is a WIP/draft PR as I get up to speed on the repo 😄

Reply:
sorry for premature feedback

Reply:
Updated to prefer yarn over npm.